specification/communication/data-plane/Common/readme.md

```diff
@@ -55,6 +55,14 @@ input-file:
   - stable/2022-07-13/common.json
 ```
 
+### Tag: package-2023-11-15-stable
```

Comment:
We need to update the tag specified at line number 29 to point to this latest version 
`tag: package-2023-11-15-stable`

Reply:
Good point, but seeing this hasn't been updated ever, can we do this in a separate PR, as a bugfix? I've already got an approval on this.

Reply:
Good point, changing the default tag.